centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { useEffect, useState } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+// import { expect } from '@storybook/jest';
+import { TextOverflowTooltip } from './TextOverflowTooltip';
+
+const meta: Meta<typeof TextOverflowTooltip> = {
+  args: {},
+  component: TextOverflowTooltip,
+  parameters: {
+    layout: 'centered'
+  }
+};
+
+export default meta;
+type Story = StoryObj<typeof TextOverflowTooltip>;
+
+export const Default: Story = {
+  args: {
+    label:
+      'Qui deserunt pariatur quis. Duis nisi velit culpa labore ipsum reprehenderit sunt laborum anim sint quis magna consequat amet. Voluptate tempor nostrud eiusmod enim qui reprehenderit.'
+  },
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  play: async ({ args, canvasElement, step }) => {
+    // FIXME
+    // const canvas = within(canvasElement);
+    // const paragraph = canvas.getByTestId('paragraph');
+    // await userEvent.hover(paragraph);
+    // await new Promise((resolve) => setTimeout(resolve, 500));
+    // const tooltip = await within(
+    //   canvasElement.parentNode as HTMLElement
+    // ).getByRole('tooltip');
+    // await expect(tooltip).toBeDefined();
+  },
```

Comment:
Should web remove or keep and fix this?

Reply:
actually the issue is with the jest + testing library setup, this code is valid otherwise
would keep it and add a description of the issue / reason in the comment

Reply:
removed it to avoid confusion